src/routes/webhooks/paddle.ts

```diff
@@ -405,6 +370,66 @@ const notifyNewPaddleTransaction = async ({
   await webhooks.transactions.send({ blocks });
 };
 
+export const processGiftedPayment = async (
+  data: TransactionCompletedEvent['data'],
+) => {
+  const con = await createOrGetConnection();
+  const { gifter_id, user_id } = data.customData as PaddleCustomData;
+
+  if (user_id === gifter_id) {
+    logger.error({ type: 'paddle' }, 'User and gifter are the same');
+    return;
+  }
+
+  const gifterUser = await con.getRepository(User).findOneBy({ id: gifter_id });
+
+  if (!gifterUser) {
+    logger.error({ type: 'paddle' }, 'Gifter user not found');
```

Comment:
log gifter_id

Reply:
Since we are now passing the `data` property, it should contain the `gifter_id` as well.